### PR TITLE
Rdm 9053 postgress11

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
RDM-9053: Postgress 11 upgrade
       [RDM-9053] (https://tools.hmcts.net/jira/browse/RDM-9053).

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -110,3 +110,56 @@ resource "azurerm_key_vault_secret" "draft-store-key" {
...
+  sku_name        = "${var.database_sku_name}"
+  sku_tier        = "GeneralPurpose"
+  sku_capacity    = "${var.database_sku_capacity}"
+  storage_mb      = "${var.database_storage_mb}"
+  common_tags     = "${var.common_tags}"
+}
+
+
+resource "azurerm_key_vault_secret" "POSTGRES-USER-V11" {
+  name         = "${var.component}-POSTGRES-USER-V11"
```
Comment (smathangi) [inline infrastructure/main.tf:138]: do we need to do phased update?? I mean create V11 database first and then migrate data and then pointing application variables to new database..?

**Review by smathangi — APPROVED** [review]
Do we really need V11 key vault entries? if not delete them.

File: infrastructure/main.tf [hunk 2]
```diff
@@ -110,3 +110,56 @@ resource "azurerm_key_vault_secret" "draft-store-key" {
...
   key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
 }
+
+////////////////////////////////
+// DB version 11             //
+////////////////////////////////
+
+module "data-store-db-v11" {
+  source          = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product         = "${var.component}-db-v11"
```
Comment (Thor-tech-of-metal) [inline infrastructure/main.tf:120]: =master needs to be confirmed with platform engineering

Comment (mario-paniccia) [inline infrastructure/main.tf:120]: can't we just use `var.product`?

**Review by mario-paniccia — APPROVED** [review]
👍 one minor comment

File: infrastructure/main.tf (hunk 2 above)
Comment (Thor-tech-of-metal) [reply infrastructure/main.tf:120]: This is going to be used for the DB name.  we would like to keep it with different values that the original DB definition module

**Pull request closed without merging**